flow-typed/aphrodite.flow.js

```diff
@@ -334,6 +334,17 @@ type _CSSProperties = {|
     ":disabled"?: _CSSProperties,
     "::-moz-focus-inner"?: _CSSProperties,
 
+    // Filters
```

Comment:
Any thoughts about including this into the valid style properties?

The main reason why I did this was to handle the separators using `:last-child` and prevent any extra logic in JS.

Reply:
I think I'd prefer that we not do this. Will comment more below...